cache/lru_cache.cc

```diff
@@ -390,7 +390,9 @@ void LRUCacheShard::Promote(LRUHandle* e) {
   assert(secondary_handle->IsReady());
   e->SetIncomplete(false);
   e->SetInCache(true);
-  e->SetPromoted(true);
+  if (!secondary_handle->IsErasedFromSecondaryCache()) {
+    e->SetPromoted(true);
```

Comment:
I think it might be better to rename "promoted" to something "IsInSecondaryCache" be prevent confusion.

Reply:
Sure.